docs/aiops-projects.md

```diff
@@ -0,0 +1,41 @@
+# AI Ops Projects 
+
+_This document contains a list of active projects within the AI Ops Team_
+
+1. **OCP Alert Prediction** : If a customer’s OpenShift cluster goes down, it can have a significant impact on their business. Since there are a variety of reasons why an OpenShift cluster might fail, finding and fixing the issue that the cluster suffers from is not always trivial. However, if we can predict in advance whether a cluster will run into a given issue, then we may be able to fix it before it fails or before it severely impacts the customer. Issues in a cluster are often defined by, or closely related to, the alerts that it fires. So predicting alerts can be a step towards predicting the underlying issue. Thus, the goal of this project is to predict whether a cluster will fire a given alert within the next hour. 
+
+2. **Categorical Encoding** : Experiments to show how to use encoding schemes for nominal categorical variables in Red Hat data with unsupervised learning problems.
```

Comment:
I would not include this as one of our active projects as it is more research based.

Reply:
Okay, I'll remove Categorical Encoding.